### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by manototh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,26 +4,53 @@
...
+       description: 'Commit ${{ github.event.head_commit.message }}' # optional
+       url: 'https://example.com' # optional, defaults to job url
+   ```
+
+1. In the code above, replace the following:
+   - Replace `DATASET_NAME` with the Axiom dataset where you want to send data.
+     To add the annotation to more than one dataset, enter a string of Axiom
+     dataset names separated by commas. For example
+     `axiom_datasets: 'DATASET_NAME_1, DATASET_NAME_2, DATASET_NAME_3'`.
+   - Replace `API_TOKEN` with your Axiom API token. Add this token to your
```
Comment (ImLunaHey) [inline README.md]: can this please be changed. this sounds like i should be putting my axiom token in the config which is not the case. maybe something more along the lines of "Please add this token to the github secrets for this repo" and remove the "Replace `API_TOKEN` with your Axiom API token." part.

File: README.md [hunk 2]
```diff
@@ -4,26 +4,53 @@
...
+
+You can configure GitHub Actions using YAML syntax. For more information, see
+the
+[GitHub documentation](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions#create-an-example-workflow).
+
+## Prerequisites
+
+- [Create an Axiom account](https://app.axiom.co/).
+- [Create a dataset](https://axiom.co/docs/reference/datasets) with the name
+  `my-dataset`.
```
Comment (ImLunaHey) [inline README.md]: > with the name `my-dataset`.

do we need this part? this works with any dataset not just ones with this name. 🤔

File: README.md [hunk 3]
```diff
@@ -4,26 +4,53 @@
...
+       title: 'Production deployment' # optional
+       description: 'Commit ${{ github.event.head_commit.message }}' # optional
+       url: 'https://example.com' # optional, defaults to job url
+   ```
+
+1. In the code above, replace the following:
+   - Replace `DATASET_NAME` with the Axiom dataset where you want to send data.
+     To add the annotation to more than one dataset, enter a string of Axiom
+     dataset names separated by commas. For example
+     `axiom_datasets: 'DATASET_NAME_1, DATASET_NAME_2, DATASET_NAME_3'`.
```
Comment (ImLunaHey) [inline README.md]: the field is `datasets` not `axiom_datasets`.

**Comment by manototh** [comment]
@ImLunaHey Thanks a lot! I've implemented your suggestions.

**Review by ImLunaHey — APPROVED** [review]
looks good 🚢 it

**Pull request merged**